README.md

```diff
@@ -60,7 +59,17 @@ Once you have implemented a lower bound search for the start of the range, imple
 
 *How do you use binary search to find the lower bound of a number? How did you have to modify the binary search algorithm?*
 
+# Awnser:
+I made my code so once i hit inside the interval i start scanning up/downstream (until reaching upper lower/bound) while appending features the to a new list along the way (which is then returned).
+
 *Would anything be more difficult if the features covered ranges instead of single nucleotides (like real BED files)? What could go wrong, if anything?*
 
+# Awnser:
+I guess we could have a region overlapping each end of the interval. In that case we could fix this by adding a similar 'start <= pos < end' for both ends of the interval. 
```

Comment:
That would catch all intervals that start or end inside the query, but it wouldn't catch intervals that start before and end after. Generally, it is a bit more difficult to efficiently capture all overlapping intervals ([but it can be done](https://en.wikipedia.org/wiki/Interval_tree))